EIPS/eip-1901.md

```diff
@@ -0,0 +1,78 @@
+---
+eip: 1901
+title: Add OpenRPC Service Discovery To JSON-RPC Services
+author: Shane Jonas (@shanejonas), Zachary Belford (@belfordz)
+discussions-to: https://github.com/ethereum/EIPs/issues/1902
+status: Draft
+type: Standards Track
+category: Interface
+created: 2019-02-25
+---
+
+# Abstract
+### What is this?
+
+This is a proposal to add [OpenRPC](https://github.com/open-rpc/spec) support to existing and future JSON-RPC services by adding the method [`rpc.discover`](https://github.com/open-rpc/spec#service-discovery-method) to the projects [JSON-RPC](https://www.jsonrpc.org/specification) APIs, enabling automation and tooling.
+
+The OpenRPC Document and generated Documentation that specifies all the methods an EVM-based blockchain should implement can be found [here](https://github.com/etclabscore/ethereum-json-rpc-specification).
+
+# Motivation
+This was first proposed [here as an ECIP](https://github.com/etclabscore/ECIPs/blob/master/ECIPs/ECIP-1053.md), but the benefits of this kind of tooling is apparent across Bitcoin, Ethereum Classic, Ethereum and other JSON-RPC accessible blockchains.
```

Comment:
I think this isn't the primary motivation. It may be more fitting for a rationale section.

Reply:
Please consider the above comment.